cmd/analyzer/expected_out/acl_testing3_detailed_explain.txt

```diff
@@ -15,25 +15,25 @@ Details:
 Path is enabled; The relevant rules are:
 	Egress:
 		security group sg1-ky allows connection with the following allow rules
-			direction: outbound, id: id:152, remote: 0.0.0.0/0, local: 0.0.0.0/0, conns: protocol: all
+			id: id:152, direction: outbound, local: 0.0.0.0/0, remote: 0.0.0.0/0, conns: protocol: all
```

Comment:
```suggestion
			id: id:152, direction: outbound, local: 0.0.0.0/0, remote: 0.0.0.0/0, protocol: all
```

Reply:
Lets fix this properly in connections. Set by adding "protocol" before "all" and then indeed the "conns" is redundant
I'll do it in a followup PR.
For now I'm removing the "conns", which will look weird when all protocols are in the connection.
This is to avoid an extra PR.

Reply:
done